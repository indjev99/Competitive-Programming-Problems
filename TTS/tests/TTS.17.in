66 6
-10000 9136.29
-10000 -8460.53
-10000 9291.42
10000 5093.59
10000 -691.35
-10000 8142.19
9343.18 10000
9382.17 -10000
-4446.72 -10000
-10000 2709.75
6652.35 -10000
2641.67 -10000
566.67 10000
10000 3785.76
4958.85 -10000
10000 3364.46
-10000 9434.33
10000 -9865.55
4150.2 -10000
10000 2748.32
-10000 -199.73
4795.85 -10000
-1 10000
-10000 5880.68
10000 -8444.89
10000 -9101.91
10000 -2717.43
-3650.28 10000
-10000 1072.81
-10000 9479.09
-10000 -3539.28
10000 8190.76
-10000 7763.17
928.49 10000
2404.25 -10000
-10000 -5073.53
3699.85 10000
6472.82 10000
-5884.75 10000
-9690.48 -10000
921.74 -10000
10000 -3080.13
10000 -8565.98
4958.51 10000
10000 -6866.07
9131.53 10000
10000 -4512.65
4751.29 10000
-10000 -2834.41
-8304.98 -10000
-7609.07 10000
-1155.66 -10000
-3345.49 -10000
9974.54 -10000
-10000 -2276.83
10000 -3444.63
-10000 -9554.62
10000 -1318.89
-10000 -4163.64
10000 4726.97
10000 -8662.04
-10000 4834.18
-5158.96 10000
10000 -6922.28
-4731.56 -10000
-10000 493.49
6836.27 7298.32
1157.61 -9932.77
-7734.2 -6338.94
1969.15 9804.21
-5252.44 -8509.52
-9719.89 -2350.25
