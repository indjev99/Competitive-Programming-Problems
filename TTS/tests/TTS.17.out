7001.090000
